apps/blas/axpy/test.expect

```diff
@@ -0,0 +1,2 @@
+{"out_0": {"0": [0], "1": [0]}, "out_1": {"0": [1], "1": [2]}, "out_2": {"0": [2], "1": [4]}, "out_3": {"0": [3], "1": [6]}, "out_4": {"0": [4], "1": [8]}, "out_5": {"0": [5], "1": [10]}, "out_6": {"0": [6], "1": [12]}, "out_7": {"0": [7], "1": [14]}, "out_8": {"0": [8], "1": [16]}, "out_9": {"0": [9], "1": [18]}, "out_10": {"0": [10], "1": [20]}, "out_11": {"0": [11], "1": [22]}, "out_12": {"0": [12], "1": [24]}, "out_13": {"0": [13], "1": [26]}, "out_14": {"0": [14], "1": [28]}, "out_15": {"0": [15], "1": [30]}, "cycles": 8}
```

Comment:
This IO format is getting annoying to read with these matrix programs 😅. We should change the test harness so it can accept inputs in other formats too!

Reply:
yeah big agree on this lol